drivers/public_reports/app/models/public_reports/homeless_population.rb

```diff
@@ -0,0 +1,576 @@
+      end
+      { data: [x, y], change: z, title: title }.to_json
+    end
+
+    private def pit_family_hoh_chart(population)
+      return {}.to_json unless population == :adults_with_children
+
+      title = 'Families'
+      x = ['x']
+      y = [title]
+      z = ['Percent change from prior quarter']
+      pit_counts(:hoh_from_adults_with_children).each do |date, counts|
+        x << date
+        y << counts[:count]
+        z << counts[:change]
+      end
+      { data: [x, y], change: z, title: title }.to_json
+    end
+
+    private def pit_counts(population)
+      data = quarter_dates.map do |date|
+        [
+          date.iso8601,
+          {
+            count: client_count_for_date(date, population),
+          },
+        ]
+      end.to_h
+      data.each_with_index do |(date, counts), i|
+        change = 0
+        if i.positive? && counts[:count].positive?
+          prior_count = data[quarter_dates[i - 1].iso8601][:count]
+          change = (((counts[:count] - prior_count.to_f) / counts[:count].to_f) * 100).round
+        end
+        data[date][:change] = change
+      end
+      data
+    end
+
+    private def client_count_for_date(date, population)
+      scope = with_service_in_quarter(report_scope, date, population).
+        select(:client_id).
+        distinct
+      # NOTE age calculations need to be done for the day in question
+
+      scope_for(population, date, scope).count
+    end
+
+    private def scope_for(population, date, scope)
+      case population
+      when :homeless
+        scope = scope.homeless
+      when :housed
+        scope = scope.permanent_housing
+      when :individuals
+        scope = scope.where(she_t[:household_id].not_in(adult_and_child_household_ids(date)))
+      when :adults_with_children
+        scope = scope.where(household_id: adult_and_child_household_ids(date))
+      when :hoh_from_adults_with_children
+        scope = scope.where(household_id: adult_and_child_household_ids(date)).heads_of_households
+      when :veterans
+        scope = scope.veterans
+      end
+      scope
+    end
+
+    private def adult_and_child_household_ids(date)
+      report_scope.service_on_date(date).where.not(household_id: nil).
+        group(:household_id).
+        having(nf('count', [shs_t[:age].gteq(18)]).gt(1).and(nf('count', [shs_t[:age].lt(18)]).gt(1))).
+        select(:household_id)
+    end
+
+    private def child_only_household_ids(date)
+      report_scope.service_on_date(date).where.not(household_id: nil).
+        group(:household_id).
+        having(nf('count', [shs_t[:age].gteq(18)]).eq(0).and(nf('count', [shs_t[:age].lt(18)]).gt(1))).
+        select(:household_id)
+    end
+
+    private def total_for(scope, population)
+      count = if population == :adults_with_children
+        scope.select(:household_id).distinct.count
+      else
+        scope.select(:client_id).distinct.count
+      end
+
+      word = case population
+      when :veterans
```

Comment:
`pluralize` is documented to take the singular, so 'Veteran', Household', and 'Person'?